docs/guide/configuration/column-options.html

```diff
@@ -95,7 +95,7 @@
 <span class="token function-variable function">formatFn</span><span class="token punctuation">:</span> <span class="token keyword">function</span><span class="token punctuation">(</span><span class="token parameter">value</span><span class="token punctuation">)</span> <span class="token punctuation">{</span>
   <span class="token keyword">return</span> <span class="token string">'$'</span> <span class="token operator">+</span> value<span class="token punctuation">;</span>
 <span class="token punctuation">}</span>
-</code></pre></div><h2 id="html"><a href="#html" aria-hidden="true" class="header-anchor">#</a> html</h2> <p>type <code>Boolean</code></p> <p>indicates whether this column will require html rendering.</p> <div class="tip custom-block"><p>The preferred way of creating columns that have html is by <a href="#custom-row-template">using slots</a></p></div> <div class="language-javascript extra-class"><pre class="language-javascript"><code><span class="token comment">// in data</span>
+</code></pre></div><h2 id="html"><a href="#html" aria-hidden="true" class="header-anchor">#</a> html</h2> <p>type <code>Boolean</code></p> <p>indicates whether this column will require html rendering.</p> <div class="tip custom-block"><p>The preferred way of creating columns that have html is by <a href="../advanced/#custom-row-template">using slots</a></p></div> <div class="language-javascript extra-class"><pre class="language-javascript"><code><span class="token comment">// in data</span>
```

Comment:
@MtDeity this is the output. which will be overwritten on a new build. you need to edit the source files.

`/vue-good-table/vp-docs/guide/configuration/column-options.md`
```
The preferred way of creating columns that have html is by [using slots](../advanced/#custom-row-template)
```